### DISCUSSION TITLE
Mid summer decorations

### DISCUSSION THREAD

**Discussion by ghost** [chromiecraft/chromiecraft#3658] [Suggestions]
Personally, I feel like Mid summer could be felt in Capital cities a bit better with more NPCs, Bonefires and dancing. There's also a lot of cute decorations that could be put into more buildings to really show the event's role play.

**Comment by Amrx2** [comment]
Hi hanabi5, the decorations and midsummer additions are all handled client-side. There will be patches out there that will allow you to change the festival's look and feel and add decorations I'm sure, these aren't supported by ourselves however.

If there are less NPCs than there should be, pop a report in and we can have a look into it but our goal is to have it as it was during 3.3.5. 

Thanks,
Construct.

**Reply by Stump876** [reply]
Already reported. The commoners are missing fron all cities. https://github.com/chromiecraft/chromiecraft/issues/3650
